osu.Game/Online/Chat/MessageFormatter.cs

```diff
@@ -279,7 +279,8 @@ public enum LinkAction
         JoinMultiplayerMatch,
         Spectate,
         OpenUserProfile,
-        Custom
+        Custom,
+        OpenDirectWithSearch
```

Comment:
this should be renamed too.

Reply:
is `SearchBeatmap` good?

Reply:
`SearchBeatmapSet`. (renamed my comment)